### DISCUSSION TITLE
Please Help with optimizing config

### DISCUSSION THREAD

**Discussion by BiAn-kisstheme** [swar/Swar-Chia-Plot-Manager#581] [Q&A]
Equipments:
SSD : M.2 NVMe SSD 1T *4 (Un-popular Brand) 

Mem : 64G (32g 3600Ghz *2)

Cpu: AMD Ryzen 9 5900X (12C 24T)

motherboard: GIGABYTE X570 GAMING X

**Comment by BiAn-kisstheme** [comment]
check_interval: 60
  log_level: ERROR

 max_concurrent: 12
  max_for_phase_1: 4
  minimum_minutes_between_jobs: 10

  - name: SSD1
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: D:\plot
    temporary2_directory:
    destination_directory: F:\plot
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 3800
    max_concurrent: 3
    max_concurrent_with_start_early: 4
    initial_delay_minutes: 0
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

  - name: SSD2
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: E:\plot
    temporary2_directory:
    destination_directory: I:\plot
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 3800
    max_concurrent: 3
    max_concurrent_with_start_early: 4
    initial_delay_minutes: 5
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

  - name: SSD3
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: J:\plot
    temporary2_directory:
    destination_directory: F:\plot
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 3800
    max_concurrent: 3
    max_concurrent_with_start_early: 4
    initial_delay_minutes: 10
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

  - name: SSD4
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: L:\plot
    temporary2_directory:
    destination_directory: I:\plot
    size: 32
    bitfield: true
    threads: 3
    buckets: 128
    memory_buffer: 3800
    max_concurrent: 3
    max_concurrent_with_start_early: 4
    initial_delay_minutes: 15
    stagger_minutes: 60
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

**Reply by jsonwu5** [reply]
How many plots will this configuration produce per day?

**Reply by BiAn-kisstheme** [reply]
28

**Comment by BiAn-kisstheme** [comment]
@Lyushen
